cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -82,6 +90,26 @@ func runCreateManagedJob(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
+	if options.manager {
+		mcid, clusterName, _, err := ocm.DefaultOCMInterface.GetManagingCluster(bpCluster.ClusterID)
+		if err == nil {
+			bpCluster, err = utils.DefaultClusterUtils.GetBackplaneCluster(mcid)
+			if err != nil {
+				return err
+			}
+		} else {
+			c, err := ocm.DefaultOCMInterface.GetClusterInfoByID(bpCluster.ClusterID)
+			if err != nil {
+				return err
+			}
+			p, ok := c.GetProduct()
+			if !ok {
+				return fmt.Errorf("could not get product information")
+			}
+			return fmt.Errorf("product id is %s and bplane url is %s for cluster: %s\nThe feature is not available for OSD and ROSA", p.ID(), bpCluster.ClusterURL, clusterName)
```

Comment:
I think this error is not entirely correct - as the not-supported combination would be an OSD/classic ROSA cluster, when using an environment that is not production (so either staging or integration).

Maybe the error could just indicate 'management cluster or shard not found in this OCM environment'.